core/modules/layout/layout.module

```diff
@@ -564,13 +572,22 @@ function layout_permission() {
       'description' => t('Add and modify layouts on pages.'),
       'restrict access' => TRUE,
     ),
+    'use all templates' => array(
+      'title' => t('Use disabled templates'),
```

Comment:
I removed the permissions in my approach over at https://github.com/backdrop/backdrop/pull/1346 so this change was not necessary.